### ISSUE TITLE
Unable to logout

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#588]
User has reported unable to log out of site. I imagine most people don't bother (I don't usuallly). I get the same issue however if I try to log out. Restarting the computer seems to have logged me out. see schreenshot of error message
<img width="883" alt="Screenshot 2024-06-17 at 09 14 33" src="https://github.com/diamm/diamm/assets/10816169/3bcc2ae1-f26e-440a-817b-72d66391923d">

**Comment by mscuthbert** [comment]
Why would anyone ever want to log out of DIAMM?  Did they learn everything they ever needed to know about medieval music?  :-D 

My guess is there was supposed to be a POST request somewhere but we're just using a GET?

**Comment by juliacmcf** [comment]
He was mostly worried that he was clogging up the system by remaining logged in. Andrew said there was a comma missing somewhere and it’s now fixed anyway!
Nice to see you Myke — how are you doing?

**Comment by ahankinson** [comment]
Yeah, apparently Django changed their logout mechanism to only accept POST requests for logging out. 

https://docs.djangoproject.com/en/5.0/releases/4.1/#log-out-via-get

**Comment by mscuthbert** [comment]
Doing great Julia!  And thanks for the update Andrew -- eek, that was going to bite another project of mine if I didn't know about it.  Thanks.  But sigh, purity over practicality was never a Django motto before ("perfectionists with deadlines") -- the original proposal 14 years ago was: when receiving a GET return a form that gives an "Are you sure?" with a POST to make the transition seamless...

Elina and I are doing well in Honolulu.  Andrew -- great work reported of yours at MEI Denton TX last month.  Was a fun conference!  Let's do a conf in Hawaii sometime...after I get a steady job, etc.

**Comment by ahankinson** [comment]
Fixed in upcoming release.
